ci_framework/roles/libvirt_manager/templates/ci-network-data.yml.j2

```diff
@@ -0,0 +1,50 @@
+{#
+The ci_job_networking is exposed by the reproducer role. It namely exposes
+the CI job configuration passed down to zuul so that we can ensure we're
+using the same networks.
+#}
+{% set ns = namespace(
+    ip4='',
+    gw4='',
+    iface=cifmw_reproducer_private_nic,
+    config_nm=false, ip_suffix=0)
+-%}
+{% if vm_type == 'controller' -%}
+{%   set ns.gw4 = cifmw_reproducer_ctl_gw4 -%}
+{%   set ns.ip4 =  ci_job_networking['instances']['controller']['networks']['default']['ip'] %}
+{% elif vm_type == 'crc' -%}
+{%   set ns.ip_suffix = 10 -%}
+{%   set ns.iface = cifmw_reproducer_crc_private_nic -%}
+{%   set ns.gw4 = cifmw_reproducer_crc_gw4 -%}
+{%   set ns.config_nm = true -%}
+{%   set ns.ip4 =  ci_job_networking['instances']['crc']['networks']['default']['ip'] %}
+{% else -%}
+{%   set ns.ip_suffix = 100+vm_id -%}
+{%   set ns.ip4 = ci_job_networking['instances']['compute-' ~ vm_id]['networks']['default']['ip']%}
+{%   set ns.gw4 = cifmw_reproducer_ctl_gw4 -%}
+{% endif -%}
+{% set default_mtu = ci_job_networking.networks['default'].mtu | default('1500') -%}
+{% set vlan_mtu = default_mtu - 4 -%}
+crc_ci_bootstrap_networks_out:
+  {{ vm_type }}-{{ vm_id }}:
+    default:
+      iface: {{ ns.iface }}
+      connection: ci-private-network
+      ip4: {{ ns.ip4 }}/24
+      gw4: {{ ns.gw4 }}
+      dns4: {{ ci_job_networking['instances']['crc']['networks']['default']['ip'] }}
+      mtu: {{ ci_job_networking.networks['default'].mtu | default(default_mtu) }}
```

Comment:
You are repeating [line](https://github.com/openstack-k8s-operators/ci-framework/pull/720/files#diff-502d2f5bb49aa7163735ee7df5f5fde39b85c21f1d5f4cc562970b338692cc9dR26) here.